src/test/java/com/coherent/testng/CartTNG.java

```diff
@@ -0,0 +1,63 @@
+package com.coherent.testng;
+
+
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Ignore;
+import org.testng.annotations.Test;
+import org.testng.asserts.Assertion;
+import org.testng.asserts.SoftAssert;
+import shop.Cart;
+import shop.RealItem;
+import shop.VirtualItem;
+
+public class CartTNG {
+
+    final double TAX = 0.2;
+    Cart cart = new Cart("UserCart");
+    RealItem realItem1;
+    VirtualItem virtualItem;
+    String cartName = "UserCart";
+
+
+    @BeforeMethod
+    public void createCard() {
+        realItem1 = new RealItem();
+        realItem1.setPrice(20);
+        realItem1.setWeight(40);
+        realItem1.setName("Item 1");
+        cart.addRealItem(realItem1);
```

Comment:
It is better to create new cart and add realItem
In @AfterMethod it is better to delete all Cart
In such case you tests will not depend on issue removing total price